### ISSUE TITLE
Specify line wrapping rules for markdown files

### ISSUE THREAD

**Issue by damithc** [oss-generic/process#83]
I'm wondering if line wrapping is necessary for markdown files (or any files for that matter). Apparently, Github can show the code with soft wraps and I suppose IDEs can do this too? In that case, inserting hard line breaks seem like a waste of effort.

![image](https://cloud.githubusercontent.com/assets/1673303/24322597/798b10c6-11a2-11e7-9dd2-8e7ea64163e4.png)

**Comment by yamgent** [comment]
Yes, in fact I feel that inserting hard line break should be discouraged for markdown documentation. Adding new words or removing words will make the wrapping out of sync, and correcting it will just add noise to the diff.

**Comment by damithc** [comment]
> Yes, in fact I feel that inserting hard line break should be discouraged for markdown documentation. Adding new words or removing words will make the wrapping out of sync, and correcting it will just add noise to the diff.

I feel the same way.

I wish we could do the same in more places, such as commit message body and javadoc comments (basically, places where we use prose).
